.github/workflows/codeql.yml

```diff
@@ -5,6 +5,8 @@ on:
     branches: [main]
   pull_request:
     branches: [main]
+  paths-ignore:
```

Comment:
What does this change do?

Reply:
Same as the linter, removing imported ibc from codeql